internal/view/drain_dialog.go

```diff
@@ -26,7 +26,13 @@ func ShowDrain(view ResourceViewer, path string, defaults dao.DrainOptions, okFn
 		SetLabelColor(styles.K9s.Info.FgColor.Color()).
 		SetFieldTextColor(styles.K9s.Info.SectionColor.Color())
 
-	var opts dao.DrainOptions
+	opts := dao.DrainOptions{
```

Comment:
Good catch Felipe!
I think we should just use the options passed into the dialog here. 
So change `defaults` to `opts` so we pick up the defaults and I think we will be good.

Reply:
updated to leverage it from args👍